Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -547,4 +547,13 @@ int Scrollbar::minimumThumbLength() const
     return m_scrollableArea.scrollbarsController().minimumThumbLength(m_orientation);
 }
 
+void Scrollbar::updateScrollbarThickness()
+{
+    if (!isCustomScrollbar()) {
+        int thickness = ScrollbarTheme::theme().scrollbarThickness(widthStyle());
+        setFrameRect(IntRect(0, 0, thickness, 0));
```

Comment:
Do we expect a layout to compute the final frame rect?

Reply:
Yeah exactly. We force a layout via `scrollableArea().scrollbarStyleChanged()`.